python/lsst/obs/pfs/isrTask.py

```diff
@@ -131,6 +132,17 @@ class PfsIsrTaskConfig(ipIsr.IsrTaskConfig):
     windowed = pexConfig.Field(dtype=bool, default=False,
                                doc="Reduction of windowed data, for real-time acquisition? Implies "
                                "overscanFitType=MEDIAN")
+    darkBBoxes = pexConfig.DictField(
+        keytype=str, itemtype=str,
+        default=dict(r3="[geom.BoxI(geom.PointI(4045, 2660), geom.PointI(4095, 2910))]"),
+        doc="List of BBoxes specifying pixels used to determine amplitude of dark frame")
+    fitDarkClipPercentiles = pexConfig.ListField(dtype=float, default=[0, 5],
+                                                 doc="""\
```

Comment:
Shouldn't need the backslash here, right?

Reply:
Otherwise there's a leading `\n`